src/dashboard/src/pages/ChainCode/ChainCode.js

```diff
@@ -335,14 +335,14 @@ class ChainCode extends PureComponent {
             <Divider type="vertical" />
             <a>
               {intl.formatMessage({
-                id: 'app.operator.chainCode.table.operate.approve',
+                id: 'app.chainCode.table.operate.approve',
```

Comment:
Can't find "app.chainCode.table.operate.approve" in locales

Reply:
it's added in #572, which may need to be reviewed and merged first.

I created two pr to make the purpose of each task more clear.

Reply:
OK